### PR TITLE
 add timestamp class test

### PR DESCRIPTION
**Pull request by hewonjeong**
add timestamp class test

### CODE REVIEW

File: src/test/node/items/models.js [hunk 1]
```diff
@@ -33,3 +33,15 @@ test('make alive key stream', t => {
 
   t.end();
 });
+
+test('make timestamp using module', t => {
+  const expected = {
+    stringResult: MAX_TIME - Number(new Date(mockData.date)),
+    dateResult: MAX_TIME - Number(new Date())
+  };
+  let timestamp = new Timestamp(mockData.date).getTimestamp();
```
Comment (ovekyc) [inline src/test/node/items/models.js]: Please use 2 const variable

**Comment by ovekyc** [comment]
LGTM

File: src/test/node/items/models.js [hunk 2]
```diff
@@ -1,11 +1,13 @@
...
-import {KeyMaker, ALIVE, DEFAULT_PRECISON, MAX_TIME} from '../../../server/items/models';
+import {KeyMaker, ALIVE, DEFAULT_PRECISON, MAX_TIME, Timestamp, GEOHASH_START_POS, GEOHASH_END_POS,
+        REF_GEOHASH_START_POS, STATUS_CODE_POS} from '../../../server/items/models';
 
 const mockData = {
   lat: 37.565398,
   lng: 126.9907941,
-  date: '2016-10-04T04:00:00.578Z',
+  dateString: '2016-10-04T04:00:00.578Z',
+  dateObject: new Date();
```
Comment (ovekyc) [inline src/test/node/items/models.js]: nono typo

**Comment by ochanje210** [comment]
@ovekyc why r u giving l g t m on travis-failed one?

**Comment by ochanje210** [comment]
plz delete ur l g t m comments

**Pull request merged**